pulp_smash/pulp3/utils.py

```diff
@@ -76,9 +76,9 @@ def sync(cfg, remote, repo):
         a call report.
     """
     client = api.Client(cfg, api.json_handler)
-    return client.post(
-        urljoin(remote['_href'], 'sync/'), {'repository': repo['_href']}
-    )
+    data = {'repository': repo['_href']}
+    data.update(kwargs)
```

Comment:
Can you please include in the docstring a sentence saying that `kwargs` will contribute to `repository` data.

```py
"""
...
:param kwargs: Keyed arguments to be merged in to repository data.
"""
```